src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -87,87 +86,80 @@
     private static final double HALF_BLOCK_OFFSET = 0.5D;
 
     /**
-     * The time in ticks the lumberjack has waited already.
-     * Directly connected with the MAX_WAITING_TIME.
+     * Number of ticks to wait for tree.
      */
-    private int timeWaited = 0;
-
+    private static final int TIMEOUT_DELAY     = 10;
+    private static final int LEAVES_RADIUS     = 3;
+    private static final int ITEM_PICKUP_RANGE = 3;
+    private static final int STUCK_WAIT_TICKS  = 20;
     /**
      * Time in ticks to wait before rechecking
      * if there are trees in the
      * range of the lumberjack
      */
     private static final int WAIT_BEFORE_SEARCH = 100;
-
     /**
      * Time in ticks before incrementing the search radius.
      */
     private static final int WAIT_BEFORE_INCREMENT = 20;
-
     /**
      * The amount of time to wait while walking to items
      */
     private static final int WAIT_WHILE_WALKING = 5;
-
     /**
      * Horizontal range in which the lumberjack picks up items
      */
     private static final float RANGE_HORIZONTAL_PICKUP = 45.0F;
-
     /**
      * Vertical range in which the lumberjack picks up items
      */
     private static final float RANGE_VERTICAL_PICKUP = 15.0F;
-
     /**
      * How often should strength factor into the lumberjacks skill modifier.
      */
     private static final int STRENGTH_MULTIPLIER = 2;
-
     /**
      * How often should charisma factor into the lumberjacks skill modifier.
      */
     private static final int CHARISMA_MULTIPLIER = 1;
-
+    /**
+     * Return to chest after half a stack
+     */
+    private static final int MAX_BLOCKS_MINED = 32;
+    /**
+     * The time in ticks the lumberjack has waited already.
+     * Directly connected with the MAX_WAITING_TIME.
+     */
+    private int timeWaited = 0;
     /**
      * Number of ticks the lumberjack is standing still
      */
     private int stillTicks = 0;
-
     /**
      * Used to store the walk distance
      * to check if the lumberjack is still walking
      */
     private int previousDistance = 0;
-
     /**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)